edison/src/main/java/edu/illinois/cs/cogcomp/edison/annotators/BrownClusterViewGenerator.java

```diff
@@ -12,18 +12,16 @@
 import edu.illinois.cs.cogcomp.core.datastructures.ViewNames;
 import edu.illinois.cs.cogcomp.core.datastructures.textannotation.SpanLabelView;
 import edu.illinois.cs.cogcomp.core.datastructures.textannotation.TextAnnotation;
-import edu.illinois.cs.cogcomp.core.datastructures.textannotation.View;
 import edu.illinois.cs.cogcomp.core.io.IOUtils;
 import edu.illinois.cs.cogcomp.core.utilities.StringUtils;
 import edu.illinois.cs.cogcomp.core.utilities.configuration.ResourceManager;
 import edu.illinois.cs.cogcomp.edison.config.BrownClusterViewGeneratorConfigurator;
+import io.minio.errors.*;
```

Comment:
IMO we should have datastore throw its own Exception instances so that the client does not have to deal with Minio directly. Right now, this assumes that the minio sdk (and namespaces) would always be available transitively. Maybe for a future PR.